opencl/extensions/public/cl_gl_private_intel_structures.h

```diff
@@ -12,24 +12,24 @@
 
 // Used for creating CL resources from GL resources
 typedef struct _tagCLGLResourceInfo {
+    GMM_RESOURCE_INFO *pGmmResInfo; /// Pointer to GMMResInfo from GL that will be copied in CL (GL)
     GLuint name;
     GLenum target;
     unsigned int globalShareHandle;
-    GMM_RESOURCE_INFO *pGmmResInfo; /// Pointer to GMMResInfo from GL that will be copied in CL (GL)
     GLenum glFormat;
     GLint glInternalFormat;
     GLuint glHWFormat;
-    GLboolean isAuxEnabled;
     GLuint borderWidth;
     GLint textureBufferWidth;
     GLint textureBufferSize;
     GLint textureBufferOffset;
+    GMM_RESOURCE_INFO *pGmmResInfoMCS;
+    GLvoid *pReleaseData;
     GLboolean oglSynchronized;
+    GLboolean isAuxEnabled;
     GMM_STATUS status;
```

Comment:
this is struct shared with GL, we have it in a form that GL is expressing. This cannot be optimized

Reply:
fix it https://github.com/intel/compute-runtime/pull/786/commits/363997de5d27e8f2e5f2271b85300ba8cb9dec7d